components/public-api/typescript/package.json

```diff
@@ -28,34 +28,16 @@
   "scripts": {
     "build": "yarn run build:cjs && yarn run build:esm",
     "build:cjs": "tsc",
-    "build:esm": "tsc --module es2015 --outDir ./lib/esm",
-    "watch": "leeway exec --package .:lib --transitive-dependencies --filter-type yarn --components --parallel -- tsc -w --preserveWatchOutput",
-    "test": "mocha './**/*.spec.ts' --exclude './node_modules/**' --exit",
-    "test:brk": "yarn test --inspect-brk"
-  },
-  "mocha": {
-    "require": [
-      "ts-node/register",
-      "reflect-metadata/Reflect",
-      "source-map-support/register"
-    ],
-    "extensions": [
-      "ts"
-    ],
-    "exit": true
+    "build:esm": "tsc --module es2015 --outDir ./lib/esm"
   },
   "dependencies": {
     "@connectrpc/connect-node": "1.1.2",
```

Comment:
it is unrelated, but these dependencies are not strictly required, and not necessary in case of browser for instance